ostree-compliance-mode.spec

```diff
@@ -0,0 +1,58 @@
+%global dracutdir %(pkg-config --variable=dracutdir dracut)
+
+Name:           ostree-compliance-mode
+Version:        0.1
+Release:        1%{?dist}
+Summary:        ostree-compliance-mode is a tool designed to purge protected data, leaving the system in a modifyable state in compliance with GPLv3
+
+License:        GPLv2
+URL:            https://github.com/ericcurtin/%{name}
+Source0:        %{url}/archive/%{version}/%{name}-%{version}.tar.gz
+
+BuildRequires: meson
+BuildRequires: gcc
+BuildRequires: ninja-build
+BuildRequires: ostree
+BuildRequires: rpm-ostree
+BuildRequires: python3-devel
+
+%description
+%{summary}
+
+%prep
+%autosetup
+%global debug_package %{nil}
```

Comment:
Do we need this?

`%global debug_package %{nil}`

Reply:
Without this, we seem to get the following build error. 

`RPM build errors:
    Empty %files file /builddir/build/BUILD/ostree-compliance-mode-0.1/debugsourcefiles.list
`

Reply:
The problem with resolving it this way, is it's just silencing the error rather than fixing it, I believe using this disables generation of debuginfo.